packages/flutter_tools/BUILD.gn

```diff
@@ -75,3 +75,26 @@ dart_tool("fuchsia_tester") {
     "$flutter_root/shell",
   ]
 }
+
+sdk_atom("fuchsia_asset_builder_sdk") {
```

Comment:
You're missing an import for this template.

Reply:
Done, not sure how it built in the first place.